### DISCUSSION TITLE
Compression

### DISCUSSION THREAD

**Discussion by SoNic1967** [schwabe/ics-openvpn#1593] [General]
Does it support LZ4-V2?
https://community.openvpn.net/openvpn/wiki/Compression
https://community.openvpn.net/openvpn/ticket/820

**Comment by schwabe** [comment]
yes. But you should not use compression. The ui does not offer lv4-v2 as selection as this option is typically pushed by the server.

**Comment by SoNic1967** [comment]
I control both endpoints (server at home) and plan to use asymmetrical compression at the server side.

When I travel, most of the places with free Wi-Fi have very low upload speeds.

 

Thank you!

**Reply by schwabe** [reply]
See here about https://community.openvpn.net/openvpn/wiki/VORACLE. 

Read the 2.6.0 man page. You probably will need to add allow-compression to the client and the proper push command the server side.

**Comment by SoNic1967** [comment]
I get compression errors if, in the client, I select

________________________________
